util.go

```diff
@@ -19,7 +20,13 @@ func readLines(name string) ([]string, error) {
 	if err != nil {
 		return nil, fmt.Errorf("cannot open file for reading: %w", err)
 	}
-	defer file.Close()
+
+	defer func() {
+		closeErr := file.Close()
+		if closeErr != nil {
+			log.Errorf("cannot close file %s: %s", name, closeErr)
+		}
+	}()
```

Comment:
This is fine as it is. I'd like to ask if you could consider this DRY-er suggestions:

```
        defer func() {
                if err = file.Close(); err != nil {
                        log.Errorf("cannot close file %s: %s", name, err)
                }   
        }() 
```

Reply:
I try to avoid such pattern, when you call function inside:`if err := function(); err != nil {